shuffle/move/examples/main/repl_help.ts

```diff
@@ -1,18 +1,127 @@
 // Copyright (c) The Diem Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-export const helpObject = [
-  "This is a Typescript/Javascript repl, with some helper!",
-  "To invoke any AsyncFunction, prefix the method with await",
-  "Let's start with the `devapi`.",
-  "Try running: `await devapi.transactions();`, `await devapi.modules();`",
-  "Now, let's check out `helpers`.",
-  "Try running: `await helpers.setMessageScriptFunction('hello');`",
-  "Now run `await devapi.accountTransactions();`",
-  "To find which arguments must be passed into each method in DiemHelpers look ",
-  "at “project_path”/main/helpers.ts",
-  "Codegen encapsulates all of the generated typescript from transaction-builders",
-  "being run on package main",
-  "To find which arguments must be passed into each method in ",
-  "codegen look at “project_path”/generated/diemTypes/mod.ts",
-];
+export let overview = {
+  "Intro": [
+    "This is a Typescript/Javascript repl.",
+    'There are 3 top level objects you can use: "main", "devapi", and "helpers".',
+  ],
+
+  "main": [
+    `"main" is the home of many script functions you can use to interact`,
+    `with the modules you've deployed onchain.`,
+    `Run "await help.main() for specific information and an example`,
+  ],
+
+  "devapi": [
+    `"devapi" contains all the information about the account being used.`,
+    `The information provided comes from many GET operations of the Diem Dev API.`,
+    `Run "await help.devapi() for specific information and an example`,
+  ],
+
+  "helpers": [
+    `"helpers" holds useful methods which can be used in your development`,
+    `Run "await help.helpers() for specific information and an example`,
+  ],
+
+  "Try Them Out": [
+    `Run each of these commands in the repl to see what functions each has`,
+    `ex. Run "devapi". Run "main"`,
+  ],
+
+  "Functions vs AsyncFunctions": [
+    `Each command provide an arsenal of AsyncFunctions and Functions at your disposal`,
+    `Note: To invoke any AsyncFunction, prefix the method with await`,
+    `ex. Run "await main.printWelcome()"`,
+  ],
+};
+
+export async function main(): Promise<any> {
+  console.log([
+    `"main" is the home of many script functions you can use to interact`,
+    `with the modules you've deployed onchain.`,
+    `Each script function creates a transaction that is deployed onchain and interacts`,
+    `with an associated module that you deployed`,
+    `Each of the methods of "main" are found in the project_dir/main/mod.ts.`,
+    `Feel free to add more methods in the mod.ts file and use them when running "main"`,
+  ]);
+  const answer = await ask(`Would you like an example? [y/n]: `);
```

Comment:
nit: You do this ask, example pattern in a few spots. You could dry this up further:


```typescript
return ask_for_example([
    `Run "main" in the repl`,
    `Run "await main.setMessageScriptFunction("hello blockchain")"`,
    `This will submit a transaction interacting directly with the`,
    `Message move module which has been deployed.`,
    `To see what your message is onchain, run "await main.decodedMessages()"`,
    `You should get an array with the String "\x00hello blockchain"`,
]);